cfgov/unprocessed/js/modules/util/dom-events.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * Parses the event passed by the bind shortcut to create the event listener
+ * @param   {string}   evt      The type of event to watch for
```

Comment:
No, it's a single event passed by `bindEvent` below. This is another bit borrowed from awesomeplete and I'm actually unsure why more than one event would ever be passed. I've removed this entire bit because for our purposes, I don't think we'd ever need it.